.github/workflows/build.yml

```diff
@@ -38,6 +38,11 @@ jobs:
             emmc-boot-image \
             tf-a-stm32mp
 
+          # Make the generates images available at a less deeply nested
+          # location for the artifact upload (the directory structure is
+          # preserved in the archive).
+          ln -s build/tmp/deploy/images/lxatac images
```

Comment:
I'm not sure if symlinks work for upload-artifact. Perhaps use normal ln (hard link)?

Reply:
Looks like you are correct. I've replaced the `ln -s` with a `mv` now. I could also use `cp --recursive --link` but hard linking directories is not really a thing, or is it?

Reply:
Ah, I missed that this is a directory. `mv` sounds good.